modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb

```diff
@@ -49,16 +49,8 @@ def healthcheck_s3
       http_status_code = 200
       s3_heathy = s3_is_healthy?
       unless s3_heathy
-        begin
-          http_status_code = 503
-          slack_details = {
-            class: self.class.name,
-            warning: "#{WARNING_EMOJI} Appeals API healthcheck failed: unable to connect to AWS S3 bucket."
```

Comment:
It looks like the `WARNING_EMOJI` constant was only used here and can therefore be removed.

Reply:
oh good one thanks